### ISSUE TITLE
"Keyword Args" section not recognized (DAR101 false positive)

### ISSUE THREAD

**Issue by HacKanCuBa** [terrencepreilly/darglint#192]
Consider the following function:

```python
def some_function(arg1: int, *, kwarg1: str) -> None:
    """Do something with the args.

    Args:
        arg1: first pos or kw argument.

    Keyword Args:
        kwarg1: first kw only argument.

    Returns:
        Nothing.
    """
    ...
```

`darglint` will raise a `DAR101 Missing parameter(s) in Docstring: - kwarg1`

Note that the section can be named `Keyword Args` or `Keyword Arguments`, see [napoleon docs](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/index.html#docstring-sections)

**Comment by terrencepreilly** [comment]
It's not a section according to the [Google styleguide](https://google.github.io/styleguide/pyguide.html#381-docstrings).  I don't think darglint's goal is to support any convention introduced by any third-party library.

Sure, it could be supported.  But would it be mandatory?  Under what circumstances?  What about positional arguments?  What configuration options?  I'm not working on darglint actively, but if you want to write a well-reasoned plan for including this feature, update all of the code/tests/documentation, feel free.  We could create a separate branch, and merge to the main branch once everything is copacetic.

**Comment by HacKanCuBa** [comment]
Alright, let me draft something.

The main idea could be that the Keyword Args section would be optional at first (not an error when present, not required to be present), and could be done mandatory in a future major version, unless we want strong adherence to the style guide, in which case it would remain optional.
